envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/HttpMethod.kt

```diff
@@ -1,5 +1,5 @@
 package pl.allegro.tech.servicemesh.envoycontrol.snapshot
 
 enum class HttpMethod {
-    GET, PUT, POST, DELETE, HEAD
+    GET, HEAD, POST, PUT, DELETE, OPTIONS, TRACE, CONNECT, PATCH
```

Comment:
Do we want `CONNECT`?

Reply:
`CONNECT` doesn't work in Envoy but lets keep all http method.

Reply:
I know it doesn't (that's why I'm asking 😄 ), can you elaborate more on how it might break?